### DISCUSSION TITLE
MQTT control

### DISCUSSION THREAD

**Discussion by KamilGRK** [seaky/nspanel_pro_tools_apk#139] [Q&A]
Hello, 

Can I control the display using MQTT ? I don't use HA

I would like to turn on the display when the system detects motion

**Comment by seaky** [comment]
yes of course:
topic: nspanelpro/`<devicename>`/command/device

**wake_up**
payload: 
{
  "command": "wake_up"
}

**sleep**
payload: 
{
  "command": "sleep"
}
